blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -221,6 +215,16 @@ public void trackResponse(HttpRequest request, HttpResponse response) {
 
     }
 
+    public void trackDelayedMetricsTenant(String tenantid, List<Metric> delayedMetrics) {
+        if (isTrackingDelayedMetrics) {
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metrics %s\n",tenantid);
+            for (Metric metric : delayedMetrics) {
+                logMessage += String.format("%s has collectionTime %d\n", metric.getLocator().toString(), metric.getCollectionTime());
```

Comment:
Converted to date and log in yyyy-MM-dd HH:mm:ss format.  Decided that it is too much changes and extra calculation to print the ingest time and interval of delay, which you can easily do by looking at the log timestamp and do the calculation outside the log.